common/interfaces/flash_interface/flash_struct.h

```diff
@@ -118,6 +118,7 @@ typedef struct Flash_Struct {
   uint8_t displayRotation;
   uint8_t enable_passphrase;
   uint8_t enable_log;
+  uint8_t onboarding_step;
```

Comment:
please update the MACRO definition `FLASH_STRUCT_TLV_SIZE` as we are adding a new member to `Flash_Struct`